src/plugin.js

```diff
@@ -121,6 +121,14 @@ const monitorPlayback = function(player, options) {
         resetMonitor();
       }
     });
+    healthcheck('adtimeupdate', function() {
```

Comment:
One of the reasons I'm asking is that if ads are being played in a separate video element, this won't handle that case as well.
So, would be best if we can figure out one solution to both the problems, particularly if we can avoid having it know that ads are a thing.